### DISCUSSION TITLE
Query Manager list empty upon start

### DISCUSSION THREAD

**Discussion by ygmarchi** [dbeaver/dbeaver#19532] [Q&A]
### Description

When I reopen dbeaver my Query Manager list is empty, even though the setting is to save them.

![image](https://user-images.githubusercontent.com/1649687/228542067-43a60829-1368-4be0-9fde-53c7c24dcbca.png)

### DBeaver Version

23.0.1

**Comment by LonwoLonwo** [comment]
Hello @ygmarchi 

History for the Query Manager is available only in our PRO products: https://dbeaver.com/

The setting which you provided is about log saving.
